somaiyalbakhet.json

```diff
@@ -7,6 +7,7 @@
     "record": {
         "A": [
             "75.2.60.5"
-        ]
+        ],
+        "CNAME": "somaiya.netlify.app"

```

Comment:
Can't have CNAME with A records, pick either of them.

Reply:
ok removed CNAME, I thought it has to be there after merging the main Netlify fallback A record